game-core/src/test/java/games/strategy/triplea/attachments/TriggerAttachmentTest.java

```diff
@@ -87,4 +94,39 @@ void testTriggerProductionFrontierEditChange() {
     assertTrue(allValues.stream().anyMatch(s -> s.contains("rule3") && s.contains("added")));
     assertTrue(allValues.stream().allMatch(s -> s.contains("frontier")));
   }
+
+  @Test
+  void testTriggerPlayerPropertyChange() throws MutableProperty.InvalidValueException {
+
+    final GameData gameData = bridge.getData();
+    final TriggerAttachment triggerAttachment = mock(TriggerAttachment.class);
+    final Set<TriggerAttachment> satisfiedTriggers = ImmutableSet.of(triggerAttachment);
```

Comment:
We usually prefer standard Java types over guava types if they actually exist.
In this case `Collections.singleton(triggerAttachment)` does the same thing: Creating an immutable set with only one entry.